Minor Commands Page (#25)
MoonlightCapital/MoonlightBot-docs: miscellaneous-commands/other-minor-commands.md

# Other Minor Commands

These commands are too minor to have thier own pages, so they are listed here.

## help

```text
/help [command]
```

* `command`: Shows information about a particular command.

> This command shows a list of commands, or info about a specific command.

Reviewer: Explain when the first or second happens

MoonlightBot requires the following permissions to successfully execute this subcommand:

> *No specific permissions required*

Reviewer: ```suggestion

*No specific permissions required*
```

pretty sure a blank line is needed between each of these to break

## ping

```text
/ping
```

> This command checks the latency and response time of the bot.

Reviewer: State it's useful to diagnose connection issues

MoonlightBot requires the following permissions to successfully execute this subcommand:
*No specific permissions required*

## stats

```text
/stats
```

> This commands shows information and statistics about the bot.

Reviewer: What statistics and info? What for?

/ping
```

> This command checks the latency and response time of the bot, which can be useful for dignosiong connection issues.

Reviewer: ```suggestion
This command checks the latency and response time of the bot, which can be useful for diagnosing connection issues.
```

/stats
```

> This commands shows information and statistics about the bot. It shows the uptime, sever count, memory usage, version, and branch of the bot, which all are useful information for diagnosing issues.

Reviewer: ```suggestion
This commands shows information and statistics about the bot. It shows the uptime, server count, memory usage, version, and branch of the bot, which all are useful information for visualizing MoonlightBot's well-being.
```

fixed typo, repetition
